packages/contracts-bedrock/scripts/libraries/LibCheat.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { stdJson } from "forge-std/StdJson.sol";
+import { VmSafe } from "forge-std/Vm.sol";
+
+/// @title LibCheat
+/// @author refcell
+/// @notice Library to write StateDiff output to json.
+library LibCheat {
```

Comment:
Maybe a more clear name? Open to other ideas too, feel free to resolve this comment yourself
```suggestion
library LibStateDiff {
```